.github/workflows/js-sdk.yml

```diff
@@ -0,0 +1,144 @@
+name: JS SDK
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    paths:
```

Comment:
Did not know this option was a thing

Reply:
i'm using this for the node bindings in `libxmtp` as well. i have some more optimizations in mind for `libxmtp` to help speed up the workflows using `paths`.